### DISCUSSION TITLE
blog/bot-kho-hon-voi-component-trong-figma

### DISCUSSION THREAD

**Discussion by giscus** [kmacoders/kmacoders.github.io#3] [General]
# blog/bot-kho-hon-voi-component-trong-figma

Xu hướng hiện tại là atomic mọi thứ có thể từ Web đến Design. Đó là lý do mà React, Vue, Angular ra đời. Vậy ở trong design, cụ thể là Figma, làm sao để tận dụng Component, để bớt khổ hơn với hệ thống hàm chục, hàng trăm screen ?

http://localhost:3000/blog/bot-kho-hon-voi-component-trong-figma

**Comment by kmacoders** [comment]
sdfsd
